### ISSUE TITLE
Show a summary of jobs next to statistics page

### ISSUE THREAD

**Issue by frankcheong** [domenkozar/almir#47]
This is really a nice tools, keep up the good work.

BTW, I guess it would not be too difficult to add a summary on the statistics (front page) containing summary about 7 days or even last month summary on total job for each level and the count of completion status (success, success with warning or even failed) of them. But this could be very useful to administrator so that they know if there is anything urgent waiting for their attention.

For I am sure administrator will be happy with the error columns can be shown in somewhat emphasized style like bold or in different colors. It would also be good to have a top errors for finished jobs and latest 1/few errors for current running jobs so that administrator can gets an heads up on what is going on.

**Comment by domenkozar** [comment]
Agreed, this would be lovely to have and is not too difficult to implement.
